src/Toml/Printer.hs

```diff
@@ -0,0 +1,56 @@
+module Toml.Printer
+       ( toml2Text
+       ) where
+
+import Data.HashMap.Strict (HashMap)
+import Data.Monoid ((<>))
+import Data.Text (Text)
+
+import Toml.Type (DateTime (..), Key (..), TOML (..), TableId (..), Value (..))
+
+import qualified Data.HashMap.Strict as HashMap
+import qualified Data.List.NonEmpty as NonEmpty
+import qualified Data.Text as Text
+
+tab :: Int -> Text
+tab n = Text.cons '\n' (Text.replicate (2*n) " ")
+
+toml2Text :: TOML -> Text
```

Comment:
Since this function is exported let's add Haddock documentation to it. Maybe with some small example from playground... What do you think?